docker/app/utils/logging_config.py

```
@@ -0,0 +1,104 @@
+
+
+# カスタムフォーマッターでログレベルに応じた色分けを実装
+class ColoredFormatter(logging.Formatter):
+    """カラー出力対応のログフォーマッター"""
+    LEVEL_COLORS = {
+        logging.DEBUG: None,  # coloramaが利用可能な場合に設定される
+        logging.INFO: None,
+        logging.WARNING: None,
+        logging.ERROR: None,
+        logging.CRITICAL: None,
+    }
```

Ruff (0.14.14)

[warning] Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)